ingress/controllers/nginx/examples/default/rc-default.yaml

```diff
@@ -14,11 +14,18 @@ spec:
         k8s-app: nginx-ingress-lb
         name: nginx-ingress-lb
     spec:
-      terminationGracePeriodSeconds: 60      
+      terminationGracePeriodSeconds: 60
       containers:
       - image: gcr.io/google_containers/nginx-ingress-controller:0.8.2
         name: nginx-ingress-lb
         imagePullPolicy: Always
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 80
```

Comment:
what's happening here is the readiness probe is getting proxied to default-http-backend/healthz. At this point we have 2 options:     

1. Make the default rc.yamls work behind a cloud provider lb by documenting that the rc must serve a health check on :80, either on some un-claimable url (weird) or by proxying to a health check url on the default backend
2. Document a pointer to a special rc if people want to deploy the nginx lb behind a cloud lb

we can't use another port, because that needs another node port, but the health checks at the cloud provider layer are per node port. I'm doing 2 with this pr, somewhat inadvertently. 

Thoughts on just adding a comment saying: "this is the health check serviced by the default backend, remove it..." something like that?